packages/ra-ui-materialui/src/form/SimpleForm.js

```diff
@@ -147,7 +147,6 @@ const enhance = compose(
     })),
     translate, // Must be before reduxForm so that it can be used in validation
     reduxForm({
-        destroyOnUnmount: false,
```

Comment:
Confirmed the bug fixed when we added this is still fixed with the two props below

Reply:
You mean #1613?

Reply:
Yes, #1997 also fixes #1613, so we no longer need that line.